src/Mapbender/CoreBundle/Resources/public/mapbender.geosource.js

```diff
@@ -718,6 +728,13 @@ Mapbender.Geo.SourceHandler = Class({
             return source.configuration.options.bbox;
         }
         return null;
+    },
+    'public function setLayerOrder': function(source, layerIdOrder) {
+        var self = this;
+        Mapbender.Geo.layerOrderMap["" + source.id] = $.map(layerIdOrder, function(layerId) {
+            var layerObj = self.findLayer(source, {id: layerId});
```

Comment:
Please use bind instead of creating a reference to the outer scope this

Reply:
[Done](https://github.com/mapbender/mapbender/pull/852/commits/19cb907bfc8007a3dde800a46d91abfbf0b7b2e9)